examples/3dtiles.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <title>3d-tiles example</title>
+
+    <style type="text/css">
+            html {height: 100%}
+        body { margin: 0; overflow:hidden; height:100%}
+
+            div {
+                margin : auto auto;
+                width: 100%;
+                padding: 0;
+                height: 100%
+                /*margin-top: 50vh;
+                transform: translateY(-50%);*/
+            }
+            #menuDiv {position: absolute; top:0px; margin-left: 0px;}
+
+
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="/node_modules/dat.gui/build/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="viewerDiv2"></div>
+        <script src="/examples/GUI/GuiTools.js"></script>
+        <script src="/dist/itowns.js"></script>
+        <script src="/dist/debug.js"></script>
+        <script type="text/javascript">
+            /* global itowns,document,GuiTools*/
+
+            const positionOnGlobe = { longitude: -75.65044586, latitude: 40.062840764, altitude: 100000 };
+
+            // iTowns namespace defined here
+            const viewerDiv = document.getElementById('viewerDiv');
+
+            const menuGlobe = new GuiTools('menuDiv');
+            const globe = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+            menuGlobe.view = globe;
+
+            const urlImageryLayers = [];
+            const urlElevationLayers = [];
+
+            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then(result => globe.addLayer(result));
+
+            itowns.Fetcher.json('layers/JSONLayers/IGN_MNT.json').then(result => globe.addLayer(result));
+            itowns.Fetcher.json('layers/JSONLayers/IGN_MNT_HIGHRES.json').then(result => globe.addLayer(result));
```

Comment:
Are those needed?
Try to keep examples short, so I'd suggest removing eveything not mandatory for your `3d-tiles` sample:
- elevation layer(s)
- `RealisticLighting` toggle
- `globe.addEventListener(..)`